### PR TITLE
Persistent Scoreboard

### PR DESCRIPTION
**Pull request by DeityLink**
image

Also removed the base64 image strings from the logs.

:cl:
* rscadd: The scoreboard of the previous round can now be checked at any point by clicking the Last Round Information button that is now always available in the top right (until the current round ends)

### CODE REVIEW

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -603,6 +603,9 @@ Assign your candidates in choose_candidates() instead.
 	var/probability = clamp(threat_level, 30, 100)
 	return !prob(probability)
 
+/datum/dynamic_ruleset/roundstart/extended/choose_candidates()
```
Comment (ShiftyRail) [inline code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm:606]: Uh ?

Comment (DeityLink) [reply code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm:606]: A one line fix for a runtime. Rulesets that don't actually take any candidates should return TRUE here.

**Comment by SonixApache** [comment]
https://github.com/vgstation-coders/vgstation13/pull/21993#issuecomment-474170097 dog bless you deity

**Comment by DeityLink** [comment]
> 
> 
> [#21993 (comment)](https://github.com/vgstation-coders/vgstation13/pull/21993#issuecomment-474170097) dog bless you deity

That's actually because I was looking into persistence to do #29753 that I checked that PR and found back your comment that I side-tracked myself into doing this one lol.

**Comment by DeityLink** [comment]
So I've managed to even keep the player images from a round to the next. That was quite the ride. Also reminder that the button will become properly usable on the round AFTER the one where this PR gets merged.

**Comment by DeityLink** [comment]
secret repo updated

**Pull request merged**